sites/upsun/src/administration/cli/reference.md

```diff
@@ -795,7 +795,7 @@ upsun activity:list --exclude-type '*.cron,*.backup*'
 
 * List pushes made before 15 March:
 ```
-upsun activity:list --type push --start 2015-03-15
+upsun activity:list --type push --start 2025-03-15
```

Comment:
this one is dynamically pulled from the `upsun list --format=md` so this date can't be override from there

Reply:
Created a pull request against the legacy CLI https://github.com/platformsh/legacy-cli/pull/1528